share/pot/stop.sh

```diff
@@ -29,13 +33,14 @@ _js_cpu_rebalance()
 # $1 pot name
 _js_stop()
 {
-	local _pname _pdir _epair _ip _aname
+	local _pname _pdir _epair _ip _aname _from_start
 	_pname="$1"
+	_from_start="$2"
+	_epair="$3"
 	_pdir="${POT_FS_ROOT}/jails/$_pname"
-	_epair=
 	_network_type=$( _get_pot_network_type "$_pname" )
 	if _is_pot_running "$_pname" ; then
-		if _is_pot_vnet "$_pname" ; then
+		if _is_pot_vnet "$_pname" && [ -z "$_epair" ]; then
```

Comment:
The interface name determined in here ends on "b" like in `epair15b`
I would suggest to change this to contain epair15a (like the interface that is passed in with the "-i" command line option), like in:

```sh
_epair=$(jexec "$_pname" ifconfig | grep ^epair | cut -d':' -f1)
_epair="${_epair%b}"a
```

Reply:
I also think that this should not be done when being called `_from_start` (if start didn't know the correct epair interface name, detecting it will probably make matters worse).

Reply:
good catch, I'm going to fix the code to always consider `epairNNa`

If `_from_start` is set, but there is no `_iface`, it's very likely that the pot is not running, so it shouldn't enter here.
However, I can add a `[ -z "$_from_start" ]` to avoid surprises